README.md

```diff
@@ -155,7 +155,7 @@ sudo touch /etc/nginx-agent/agent-dynamic.conf
 ```
 
 ### Enabling the gRPC interface
-Update the `/etc/nginx-agent/nginx-agent.conf` file to include the following settings:
+Add the the following settings to `/etc/nginx-agent/nginx-agent.conf`:
```

Comment:
I think there's a typo. The line seems to read "Add the the..." now

Reply:
```suggestion
Add the following settings to `/etc/nginx-agent/nginx-agent.conf`:
```